src/Storage/Commands.Storage/Common/Cmdlet/DisableAzureStorageStaticWebsite.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
+{
+    using Microsoft.WindowsAzure.Commands.Storage.Model.ResourceModel;
+    using Microsoft.WindowsAzure.Storage.Shared.Protocol;
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+
+    /// <summary>
+    /// Disable azure storage service StaticWebsite, currently only available on Blob service
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Disable, StorageNouns.ServiceStaticWebsite, SupportsShouldProcess = true),
+        OutputType(typeof(PSStaticWebsiteProperties))]
+    public class DisableAzureStorageServiceStaticWebsiteCommand : StorageCloudBlobCmdletBase
+    {
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
+
+        // Overwrite the useless parameter
```

Comment:
What are these about? to set them in the tests?

Reply:
This parameter will show the user the StaticWebsite setting after the change. So they can have a check.
This is aligned with other serverproperty cmdlets like
 
enable/disable-AzureStorageDeleteRetentionPolicy
Set-AzureStorageServiceLoggingProperty
Set-AzureStorageServiceMetricsProperty
Set-AzureStorageServiceProperty

Reply:
The comment says "Overwrite the useless parameter".. these dont seem useless :)